serverless.yml

```diff
@@ -1,8 +1,13 @@
 service: sls-data-service
 
+custom:
+  MONGO_URI: mongodb://127.0.0.1:27017/gorditos
```

Comment:
Doea this force us to install mongo locally?

Reply:
I am not sure here. I was told we were supposed to use a local mongo db.
How can I get a connection string for the correct database?

Reply:
You can set your env before the `sls offline` instead of putting them in the yml